src/partials/user.hbs

```diff
@@ -0,0 +1,5 @@
+{{#each .}}
+<p><strong>{{@key}}</strong> - {{userId}}</p>
```

Comment:
Instead of P tag use span/div which makes more sense.

Reply:
this file is not being used